packages/block-editor/src/components/list-view/use-list-view-scroll-into-view.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * WordPress dependencies
+ */
+import { getScrollContainer } from '@wordpress/dom';
+import { useLayoutEffect } from '@wordpress/element';
+
+export default function useListViewScrollIntoView( {
+	isSelected,
+	selectedClientIds,
+	rowItemRef,
+} ) {
+	const numBlocksSelected = selectedClientIds?.length;
```

Comment:
Nit: Let's rename this variable into something like `isMultiSelected` so that we don't have to needlessly rerun the effect when the number of selected blocks changes.

Reply:
Good call. I changed the name to `isSingleSelection` because I think that gets slightly closer to the existing check, but either name would work 🙂